### DISCUSSION TITLE
Is the Vercel Data Cache available to other frameworks?

### DISCUSSION THREAD

**Discussion by richardscarrott** [vercel/community#3757] [Feature Question] [answered]
### Question

I'm interested in using the [Vercel Data Cache](https://vercel.com/docs/infrastructure/data-cache) in a Remix codebase.

My understanding is the [cache extension of `window.fetch`](https://vercel.com/docs/infrastructure/data-cache#examples) is done by Nextjs, however I'm guessing it's powered using [Cloudflare Workers underlying `window.caches` API](https://developers.cloudflare.com/workers/runtime-apis/cache/#accessing-cache)?

I wondered if either API is available to applications other than NextJS?

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out, @richardscarrott. It is currently limited to Next.js, but I let the team know you're interested in using it with Remix.
